src/components/Alert/index.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+import { ReactComponent as AlertIcon } from 'assets/images/icons/alert.svg';
+
+interface AlertProps {
+  errors: string[];
+  'data-test-id'?: string;
+}
+const Alert = ({ errors, ...rest }: AlertProps): JSX.Element => (
+  <div role="alert" {...rest}>
+    <div className="bg-black-raisin bg-opacity-60 rounded-xl px-4 py-4 flex flex-row">
```

Comment:
Could you recheck the form layout and this Alert a bit?
It seems the current UI does not match the design:
|Design|Current|
|-|-|
|![image](https://github.com/manh-t/react-survey/assets/7344405/c17f3ca9-9e83-4d6d-a1a4-58fbd94acc03)|![SCR-20230718-kpbp](https://github.com/manh-t/react-survey/assets/7344405/269ad4d8-39e4-4ced-87e5-a78461a02949)|

Reply:
@hoangmirs okay, let me fix the alert width 😅

Reply:
@hoangmirs fixed in [90287e8](https://github.com/manh-t/react-survey/pull/49/commits/90287e88fb7a8b870a60719bb83fc9a8264dfcea)